core/lib/components/balena/sdk.js

```diff
@@ -418,4 +447,99 @@ module.exports = class BalenaSDK {
 				.get('current_services'),
 		);
 	}
+
+	/** Pushes a release to an application, from a given directory 
+	 * @param application the balena application name
+	 * @param directory the directory name containing the docker-compose for the application and the source files
+	*/
+	async pushReleaseToApp(application, directory){
+		await exec(
+			`balena push ${application} --source ${directory}`
+		  );
+		//check new commit of app
+		let commit = await this.balena.models.application.get(
+			application
+		)
+		.get("commit");
```

Comment:
```suggestion
		let commit = await this.balena.models.application.get(application).get("commit");
```
Did prettify do this?